traffic_ops/app/db/seeds.sql

```diff
@@ -175,6 +175,9 @@ insert into capability (name, description) values ('server-capabilities-write',
 -- servers
 insert into capability (name, description) values ('servers-read', 'Ability to view servers') ON CONFLICT (name) DO NOTHING;
 insert into capability (name, description) values ('servers-write', 'Ability to edit servers') ON CONFLICT (name) DO NOTHING;
+-- service categories
+insert into capability (name, description) values ('service-categories-read', 'Ability to view service categories') ON CONFLICT (name) DO NOTHING;
+insert into capability (name, description) values ('service-categories-write', 'Ability to edit service categories') ON CONFLICT (name) DO NOTHING;
```

Comment:
I think you meant to insert service categories here?

Reply:
This is adding service category capability so it should be the capability table